packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -0,0 +1,210 @@
+import { 
+  RDSClient,
+  CreateDBInstanceCommand,
+  waitUntilDBInstanceAvailable,
+  DeleteDBInstanceCommand,
+  waitUntilDBInstanceDeleted 
+} from "@aws-sdk/client-rds";
+import { EC2Client, AuthorizeSecurityGroupIngressCommand, RevokeSecurityGroupIngressCommand } from '@aws-sdk/client-ec2';
+import { knex } from 'knex';
+
+const DEFAULT_DB_INSTANCE_TYPE = "db.m5.large";
+const DEFAULT_DB_STORAGE = 8;
+const DEFAULT_SECURITY_GROUP = "default";
+
+export const createRDSInstance = async (config: {
```

Comment:
Could you add docs to these new utils. No time like the present to improve the state of our docs 👯

Reply:
Could you also add response types to the declaration?

Reply:
Added docs for the new methods.